pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -231,11 +232,16 @@ func AssignFlavors(log logr.Logger, wl *workload.Info, resourceFlavors map[kueue
 		usage:       make(cache.FlavorResourceQuantities),
 	}
 	for i, podSet := range wl.TotalRequests {
+		if _, found := cq.RGByResource[constants.NumPodsResourceName]; found {
+			podSet.Requests[constants.NumPodsResourceName] = int64(wl.Obj.Spec.PodSets[i].Count)
+		}
```

Comment:
Do you have any reasons why the FlavorAssignment (scheduler) updates the snapshot rather than the cache updates the ClusterQueue?

This means I think we should update the cache instead of updating the snapshot in the scheduler.

Reply:
I see. Thank you for clarifying!